.github/workflows/trivy.yml

```diff
@@ -15,29 +15,43 @@ jobs:
       fail-fast: false
       matrix:
         include:
-        - dir: batch
-        - dir: bastion
-        - dir: cis
-        - dir: sdarq/frontend
-        - dockerfile: sdarq/backend/Dockerfile
-        - dir: zap
+          - dir: batch
+          - dir: bastion
+          - dir: cis
+          - dir: defectdojo
```

Comment:
This doesn't work. The defectdojo scan keeps failing because the Dockerfile is named funny (Dockerfile.django). There will need to be changes to the build files in order to fix this.

Reply:
Right. I took it back out yesterday but neglected to push the change. It could also be changed in the trivy action but I'm leaving it out for now.